build/README.md

```diff
@@ -23,16 +23,16 @@ There is also early support for building Docker "run" containers
 
 The following scripts are found in the `build/` directory:
```

Comment:
maybe add a sentence here noting that these scripts must be run from the Kubernetes root directory?